tests/js/client/aql/aql-index-batch-materialize.js

```diff
@@ -182,6 +192,47 @@ function IndexBatchMaterializeTestSuite() {
       expectNoOptimization(query);
     },
 
+    testMaterializeIndexScanProjections: function () {
+      const query = `
+        FOR doc IN ${collection}
+          FILTER doc.x > 5
+          RETURN [doc.y, doc.z, doc.a]
+      `;
+      const {materializeNode, indexNode} = expectOptimization(query);
+      assertTrue(indexNode.projections, undefined);
```

Comment:
I can't make sense of this assertion.
Should it rather be
```suggestion
      assertUndefined(indexNode.projections);
```